tests/Predis/Command/Redis/Search/FT_LIST_Test.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2025 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Redis\Search;
+
+use Predis\Command\Argument\Search\SchemaFields\TextField;
+use Predis\Command\Redis\PredisCommandTestCase;
+
+class FT_LIST_Test extends PredisCommandTestCase
+{
+    /**
+     * {@inheritDoc}
+     */
+    protected function getExpectedCommand(): string
+    {
+        return FT_LIST::class;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function getExpectedId(): string
+    {
+        return 'FT_LIST';
```

Comment:
Should this be `FT._LIST` instead of `FT_LIST` ?

Reply:
Not really, this one corresponds to class name rather then command ID, a bit confusing but that's how it works